src/pages/boundary/api-docs/[[...page]].tsx

```diff
@@ -92,9 +106,19 @@ export async function getStaticProps({ params }) {
 	// Construct sidebar nav data levels
 	const sidebarNavDataLevels = [
 		{
+			backToLinkProps: { text: 'Boundary Documentation', href: '/boundary/' },
 			menuItems: [
 				{
-					heading: apiPageProps.info.title,
+					heading: 'API',
+				},
+				{
+					title: 'Overview',
+					indexData: true,
+					path: '',
+					fullPath: '/boundary/api-docs/',
+				},
```

Comment:
I'm not sure what `indexData` is here. Also, I think you only need either `path` or `fullPath`.

Reply:
Removing `indexData` and using only `fullPath`